lib/tbot/config/cli.go

```diff
@@ -0,0 +1,579 @@
+/*
+ * Teleport
+ * Copyright (C) 2024  Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package config
```

Comment:
My only "concern" here is pushing this stuff down into the already fairly over-scoped "config" package. We should keep in mind that `tbot` is seeing embedded usage now, and we're going to be inflicting any dependencies introduced into `lib/tbot` and `lib/tbot/config` on any consumer. If it's possible to have this as a separate `lib/tbot/cli` without a large amount of work, that would be nice, but otherwise we can revisit this.

Reply:
Agreed, I've moved it into its own package. And to fully commit to separating `lib/tbot/config` from CLI stuff, all kingpin related functionality is now removed from that package and the import is gone.